src/mpcCoreKit.ts

```diff
@@ -195,8 +207,8 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
       useTSS: true,
       customAuthArgs: {
         web3AuthClientId: this.options.web3AuthClientId,
-        baseUrl: this.options.baseUrl ? this.options.baseUrl : `${window.location.origin}/serviceworker`,
-        uxMode: this.options.uxMode,
+        baseUrl: this.options.baseUrl,
+        uxMode: this.isNodejsOrRN(this.options.uxMode) ? UX_MODE.REDIRECT : (this.options.uxMode as UX_MODE_TYPE),
```

Comment:
would be better to have a dedicate ux mode other thn redirect for node js/react native

Reply:
that is passed into customAuth which expecting "popup" or "redirect"
have to set to "redirect" because setting up customAuth with "popup" wiil hit some browser api which is not supported in nodejs and rn
hence force to "redirect" for nodejs and rn